### ISSUE TITLE
Create Ministries Page

### ISSUE THREAD

**Issue by austinEng** [GraceCovenantChurch/gccweb#45]
Create the static ministries page 

Coordinate with @estheryjeon on design. Initial templates are here: https://invis.io/DKDRIATQ6

**Comment by vce23** [comment]
IDK how, but I got the page running. Now I need to get access to the code itself

I don't have access to the subpages folder? (gccweb/src/client/components/subpage) Apparently I have a file open in atom there, but I can't access the folder

**Comment by austinEng** [comment]
Huh. Not aware that such a folder exists. Did you create it?

**Comment by joeyzia** [comment]
I think that's the directory I created for I'm New... Maybe some branching issues?

**Comment by joeyzia** [comment]
@heallen Finished this -- still have to change pictures and such. Can we assign it to you @estheryjeon or people on design?

**Comment by austinEng** [comment]
@estheryjeon This will also need to include background images for each tile, correct?

**Comment by estheryjeon** [comment]
@austinEng Yes! But was unsure where to find a photo of each ministry team?
